### PR TITLE
Des 550

### PR DESCRIPTION
**Pull request by JasonLantz**
event harness, Button, Icon, TextInput, PasswordInput, tests, docs.....

### CODE REVIEW

File: demo/demo.scss [hunk 1]
```diff
@@ -108,13 +108,13 @@ li {
...
 
 //=========Button Page===============================
-svg {
-  margin-left: 10px;
-}
-
-button svg {
-  margin: 4px 0 0;
-}
+//svg {
```
Comment (wyseguyonline) [inline demo/demo.scss]: Is this to be removed or just commented out for now (i.e. we will be reinstating this code at a later date)?

File: npm_scripts/exec.js [hunk 2]
```diff
@@ -1,11 +0,0 @@
-// ES6 Modules support still in progress for the V8 engine, hence using 'require' here
```
Comment (wyseguyonline) [inline npm_scripts/exec.js]: Are these npm_scripts files being replaced by the ones in our npm-scripts repo?

File: src/Icon.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ const Icon = (props) => {
            xmlns = "http://www.w3.org/2000/svg"
            xmlnsXlink = "http://www.w3.org/1999/xlink"
            role = "img"
+           focusable = "no"
```
Comment (wyseguyonline) [inline src/Icon.js]: Should focusable be configurable?

File: demo/demo.scss (hunk 1 above)
Comment (StommePoes) [inline demo/demo.scss]: Only if it's still needed to "center" the svg in the button. There's no standard set of CSS I could find and the icon-in-a-button was a demo use case.

File: src/Icon.js (hunk 3 above)
Comment (StommePoes) [inline src/Icon.js]: No. The icons themselves should never be focusable. IE is following the TinySVG 1.2 spec which was never actually approved, so no current (nor future) SVG spec allows for this attribute (which in IE is defaulted to true/yes).

Comment (wyseguyonline) [inline src/Icon.js]: Thanks for the clarification.

**Pull request merged**